pod-operation/src/components/lim_temperature.rs

```diff
@@ -19,8 +22,9 @@ const R_0: f32 = 10000.0; // Ohms
 const ROOM_TEMP: f32 = 25.0 + C_TO_K_CONVERSION; // Kelvins
 
 fn voltage_to_temp(voltage: i16) -> f32 {
-	let voltage = f32::from(voltage) / 1000.0;
-	let thermistor_resistance = (voltage * DIVIDER_RESISTANCE) / (V_IN - voltage);
+	let voltage = f32::from(voltage) / 500.0;
```

Comment:
Chore: with the change to FSR, this function no longer receives the actual milli-voltage. Could we move the LSB scaling to where the actual reading occurs (see [`lim_current.rs:38`](https://github.com/UCI-HyperXite/hx9-control-system/blob/af5c7cb0fc697535749d123f6fe1ab2354a17023/pod-operation/src/components/lim_current.rs#L38) for reference).

Reply:
wdym lsb scaling?